### PR TITLE
Creates a notebook for batch-prediction pipeline with BQ I/O on a custom-tabular model

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR creates a notebook in the notebook/official/pipelines/ folder under the name "`custom_tabular_train_batch_pred_bq_pipeline.ipynb`". The notebook mainly focuses on creating a Vertex AI pipeline for:

- Loading a tabular dataset with BigQuery source.
- Training a model using custom python package.
- Running batch prediction job on the custom model with BigQuery input and output.
<br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,1461 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>Sklearn -&gt; scikit-learn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>location="US". -&gt; location = REGION.split('-')[0]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>below code -&gt; code below</p><p>Sklearn -&gt; scikit-learn</p><p> </p><p>For more details, check <a href="https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html" target="_blank">Sklearn pipelines</a>. -&gt; use format:</p><p> </p><p>Learn more about [scikit-learn pipelines](link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>sklearn -&gt; scikit-learn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>Vertex Model Artifact -&gt; Vertex AI Model artifact</p><p> </p><p>For links, use format</p><p> </p><p>Learn more about [title](link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>remove the 'return'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: <p>avoid trailing commas</p><p> </p><p>enable_caching=False,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (krishr2d2) [reply notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: updated.

Comment (krishr2d2) [reply notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: done

Comment (krishr2d2) [reply notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb:1]: updated.

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
Looks like notebook was previous reviewed and issues resolved

**Pull request merged**